MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include atool/templates *.html
```

Comment:
Just use [`package_data`](https://packaging.python.org/en/latest/distributing.html#package-data) in `setup.py` which will be included in [`MANIFEST`](https://docs.python.org/2.7/distutils/sourcedist.html#specifying-the-files-to-distribute) so you can get rid of [`MANIFEST.in`](https://packaging.python.org/en/latest/distributing.html#manifest-in).